translations/messages+intl-icu.de.yml

```diff
@@ -1229,6 +1229,7 @@ error.userStory.voting.reset: "Ihre Bewertung konnte nicht zurückgesetzt werden
 error.userStory.voting: "Die Bewertung konnte nicht gespeichert werden."
 error.without.authorization: "Sie sind zu dieser Aktion nicht autorisiert."
 error.worksheet.name: "Bitte wählen Sie einen gültigen Titel ({expectedTitles}) für das Arbeitsblatt {worksheetTitle}."
+error.wrong.selected.importer: "Sie haben ein Segment importiert. Bitte wählen Sie die Option Abschnitte importieren und importieren Sie Ihre Datei erneut."
```

Comment:
the ticket says that the errormessage should mention the stn import and that the format is wrong... you also mention Segment - this should be called 'Abschnitt' vor users.
my proposal would be:

"Sie haben die Option Stellungnahmen importieren genutzt um Abschnitte zu importieren. Bitte wählen Sie die korrekte Option aus und importieren Sie Ihre Datei erneut."

Reply:
done [3a1aa38](https://github.com/demos-europe/demosplan-core/pull/1902/commits/3a1aa38869b386757afc154f8340ab39c0b97080)